packages/react/src/Timeline/Timeline.module.css

```diff
@@ -0,0 +1,62 @@
+.Timeline {
+  --timelineSpacing: var(--base-size-16);
+  --itemLineHeight: var(--brand-text-lineHeight-300);
+  --bulletSize: var(--base-size-8);
+  --bulletSpacing: 2px;
+  --bulletY: calc(var(--itemLineHeight) / 2 - var(--bulletSize) / 2);
+  --lineWidth: 1.5px;
+
+  margin: 0;
+  padding: 0px;
+  list-style: none;
+}
+
+.Timeline-item {
+  position: relative;
+  max-width: 44ch;
+  padding-inline-start: calc(var(--bulletSize) + var(--timelineSpacing));
+  color: var(--brand-color-text-default);
+  font-family: var(--brand-body-fontFamily);
+  font-size: var(--brand-text-size-300);
+  font-weight: var(--brand-text-weight-200);
+  line-height: var(--itemLineHeight);
+}
+
+.Timeline-item:not(:last-child) {
+  padding-bottom: var(--base-size-32);
+}
+
+.Timeline-item::before,
+.Timeline-item::after {
+  content: '';
+  position: absolute;
+}
+
+.Timeline-item::before {
+  width: var(--bulletSize);
+  height: var(--bulletSize);
+  top: var(--bulletY);
+  left: 0;
+  border: solid var(--lineWidth) var(--brand-color-border-default);
+  border-radius: 100%;
+}
+
+.Timeline-item:not(:last-child)::after {
+  top: calc(var(--bulletY) + var(--bulletSize) + var(--bulletSpacing));
+  left: calc(var(--bulletSize) / 2);
+  width: var(--lineWidth);
+  height: calc(100% - var(--bulletSize) - var(--bulletSpacing) * 2);
+  transform: translateX(-50%);
+  background-color: var(--brand-color-border-default);
+  border-radius: var(--lineWidth);
+}
+
+.Timeline-item:has(em) {
```

Comment:
`has` isn't [universally supported](https://caniuse.com/css-has) across all browsers. could you hoist this logic into the React layer for the time being. Example in this PR: https://github.com/primer/brand/pull/376/files#diff-dcc5852ace9baf504d6890f39cf2a451591d81598cbdcc9bcb4f716c8a74f69fR53-R58. Open to other approaches also.